src/_guides/language/language-tour.md

```diff
@@ -544,11 +544,10 @@ To represent boolean values, Dart has a type named `bool`. Only two
 objects have type bool: the boolean literals `true` and `false`,
 which are both compile-time constants.
 
-When Dart expects a boolean value, only the value `true` is treated as
-true. All other values are treated as false. Unlike in JavaScript,
-values such as `1`, `"aString"`, and `someObject` are all treated as
-false.
+In JavaScript many values, such as `1`, `"aString"`, and `someObject` are treated as
+true, whereas values like null, 0 or empty string are treated as false.
```

Comment:
I just realized that, with Dart 2, all of this should be unnecessary: you shouldn't be assigning non-boolean values to a boolean.

Let me check on this and get back to you.

Reply:
It looks like we need to delete everything from "In JavaScript" up to "What this means for you is that, instead". Instead, it'll be something like:

> ...compile-time constants.
> 
> Dart's type safety means that you can't use code like `if (nonbooleanValue)` or `assert(nonbooleanValue)`. Instead, explicitly check for values, like this:
----------

@leafpetersen does that look correct?

Reply:
Looks reasonable to me.

Reply:
@ArsProgramma would you like to make this change?